apps/web/src/features/safenet/components/new-safe/SafenetBanner/index.tsx

```diff
@@ -0,0 +1,71 @@
+import darkPalette from '@/components/theme/darkPalette'
+import lightPalette from '@/components/theme/lightPalette'
+import { AppRoutes } from '@/config/routes'
+import SafeLogoNoText from '@/public/images/logo-no-text-transparent.svg'
+import SafenetLogo from '@/public/images/safenet/logo-safenet.svg'
+import { Box, Button, Grid, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import css from './styles.module.css'
+
+const SafenetBanner = () => {
+  const router = useRouter()
+  const { palette } = useTheme()
+  const [displayBanner, setDisplayBanner] = useState<boolean>(true)
```

Comment:
Usually we use the local storage to remember that a user closed a banner.

```suggestion
  const [displayBanner = true, setDisplayBanner] = useLocalStorage<boolean>("showSafenetBanner")
```